CHANGELOG.md

```diff
@@ -21,6 +21,13 @@ Infusion from v2.0.0 onwards adheres to [Semantic Versioning](https://semver.org
 
 ### More Info
 
+## [4.1.0] - 2022-05-09
```

Comment:
Will you actually be cutting a release for this? If so I guess the date needs to be updated now, if not these can all go to the unreleased section until we're ready to do the release.